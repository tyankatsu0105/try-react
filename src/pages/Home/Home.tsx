import React, { FunctionComponent } from 'react';

const Home: FunctionComponent = () => {
  return <div>aaa</div>;
};

export default Home;
