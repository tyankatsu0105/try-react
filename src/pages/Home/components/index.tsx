import React, { FC } from 'react';

const Home: FC = () => {
  return <p>this is home</p>;
};

export default Home;
