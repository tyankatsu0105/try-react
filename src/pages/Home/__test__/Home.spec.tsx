import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import Home from '../Home';

describe('Home', () => {
  it('render', () => {
    const { baseElement } = render(<Home />);
    expect(baseElement).toMatchSnapshot();
  });
});
