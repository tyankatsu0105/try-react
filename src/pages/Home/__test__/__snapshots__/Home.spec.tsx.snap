// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home render 1`] = `
<body>
  <div>
    {
  "user": "",
  "email": "",
  "gender": "男性",
  "animal": "",
  "colors": [],
  "option1": false,
  "option2": false,
  "option3": true,
  "date": "2019-08-28T05:46:39.701Z"
}
    <form
      autocomplete="off"
    >
      <div
        class="MuiFormGroup-root"
      >
        <div
          class="MuiBox-root MuiBox-root-5"
        >
          <div
            class="MuiFormControl-root"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink="false"
              for="user"
            >
              ユーザー名
            </label>
            <div
              class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
            >
              <input
                aria-describedby="input-user"
                aria-invalid="false"
                class="MuiInputBase-input MuiInput-input"
                id="user"
                name="user"
                type="input"
                value=""
              />
            </div>
            <p
              class="MuiFormHelperText-root"
              id="user"
            >
              ユーザー名を入力
            </p>
          </div>
        </div>
        <div
          class="MuiBox-root MuiBox-root-67"
        >
          <div
            class="MuiFormControl-root"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink="false"
              for="email"
            >
              メールアドレス
            </label>
            <div
              class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
            >
              <input
                aria-describedby="input-email"
                aria-invalid="false"
                class="MuiInputBase-input MuiInput-input"
                id="email"
                name="email"
                type="input"
                value=""
              />
            </div>
            <p
              class="MuiFormHelperText-root"
              id="email"
            >
              メールアドレスを入力
            </p>
          </div>
        </div>
        <div
          class="MuiBox-root MuiBox-root-68"
        >
          <div
            class="MuiFormControl-root"
          >
            <legend
              class="MuiFormLabel-root"
            >
              性別
            </legend>
            <div
              aria-label="gender"
              class="MuiFormGroup-root"
              role="radiogroup"
            >
              <label
                class="MuiFormControlLabel-root"
              >
                <span
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-80 MuiRadio-root MuiRadio-colorSecondary PrivateSwitchBase-checked-81 Mui-checked MuiIconButton-colorSecondary"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <div
                      class="PrivateRadioButtonIcon-root-96 PrivateRadioButtonIcon-checked-98"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        role="presentation"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                        />
                      </svg>
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root PrivateRadioButtonIcon-layer-97"
                        focusable="false"
                        role="presentation"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                        />
                      </svg>
                    </div>
                    <input
                      checked=""
                      class="PrivateSwitchBase-input-83"
                      name="gender"
                      type="radio"
                      value="男性"
                    />
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </span>
                <span
                  class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  男性
                </span>
              </label>
              <label
                class="MuiFormControlLabel-root"
              >
                <span
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-80 MuiRadio-root MuiRadio-colorSecondary MuiIconButton-colorSecondary"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <div
                      class="PrivateRadioButtonIcon-root-96"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        role="presentation"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                        />
                      </svg>
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root PrivateRadioButtonIcon-layer-97"
                        focusable="false"
                        role="presentation"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                        />
                      </svg>
                    </div>
                    <input
                      class="PrivateSwitchBase-input-83"
                      name="gender"
                      type="radio"
                      value="女性"
                    />
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </span>
                <span
                  class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  女性
                </span>
              </label>
              <label
                class="MuiFormControlLabel-root"
              >
                <span
                  aria-disabled="false"
                  class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-80 MuiRadio-root MuiRadio-colorSecondary MuiIconButton-colorSecondary"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <div
                      class="PrivateRadioButtonIcon-root-96"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        role="presentation"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                        />
                      </svg>
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root PrivateRadioButtonIcon-layer-97"
                        focusable="false"
                        role="presentation"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                        />
                      </svg>
                    </div>
                    <input
                      class="PrivateSwitchBase-input-83"
                      name="gender"
                      type="radio"
                      value="その他"
                    />
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </span>
                <span
                  class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  その他
                </span>
              </label>
              <label
                class="MuiFormControlLabel-root Mui-disabled"
              >
                <span
                  aria-disabled="true"
                  class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-80 MuiRadio-root MuiRadio-colorSecondary PrivateSwitchBase-disabled-82 Mui-disabled MuiIconButton-colorSecondary Mui-disabled Mui-disabled"
                  tabindex="-1"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <div
                      class="PrivateRadioButtonIcon-root-96"
                    >
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root"
                        focusable="false"
                        role="presentation"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                        />
                      </svg>
                      <svg
                        aria-hidden="true"
                        class="MuiSvgIcon-root PrivateRadioButtonIcon-layer-97"
                        focusable="false"
                        role="presentation"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                        />
                      </svg>
                    </div>
                    <input
                      class="PrivateSwitchBase-input-83"
                      disabled=""
                      name="gender"
                      type="radio"
                      value="選択不可"
                    />
                  </span>
                </span>
                <span
                  class="MuiTypography-root MuiFormControlLabel-label Mui-disabled MuiTypography-body1"
                >
                  選択不可
                </span>
              </label>
            </div>
          </div>
        </div>
        <div
          class="MuiBox-root MuiBox-root-138"
        >
          <div
            class="MuiFormControl-root"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
              data-shrink="false"
              for="animal"
            >
              好きな動物
            </label>
            <div
              class="MuiInputBase-root MuiInput-root MuiInput-underline makeStyles-home-Select-1 MuiInputBase-formControl MuiInput-formControl"
            >
              <div
                aria-haspopup="listbox"
                class="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiInputBase-input MuiInput-input MuiInputBase-inputSelect"
                id="select-animal"
                role="button"
                tabindex="0"
              >
                <span>
                  ​
                </span>
              </div>
              <input
                id="animal"
                name="animal"
                type="hidden"
                value=""
              />
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSelect-icon"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 10l5 5 5-5z"
                />
              </svg>
            </div>
          </div>
        </div>
        <div
          class="MuiBox-root MuiBox-root-149"
        >
          <div
            class="MuiFormControl-root"
          >
            <label
              class="MuiFormLabel-root"
              for="colors"
            >
              好きな色
            </label>
            <label
              class="MuiFormControlLabel-root"
            >
              <span
                aria-disabled="false"
                class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-80 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                    />
                  </svg>
                  <input
                    class="PrivateSwitchBase-input-83"
                    data-indeterminate="false"
                    id="colors"
                    name="colors"
                    type="checkbox"
                    value="赤"
                  />
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                赤
              </span>
            </label>
            <label
              class="MuiFormControlLabel-root"
            >
              <span
                aria-disabled="false"
                class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-80 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                    />
                  </svg>
                  <input
                    class="PrivateSwitchBase-input-83"
                    data-indeterminate="false"
                    id="colors"
                    name="colors"
                    type="checkbox"
                    value="オレンジ"
                  />
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                オレンジ
              </span>
            </label>
            <label
              class="MuiFormControlLabel-root"
            >
              <span
                aria-disabled="false"
                class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-80 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                    />
                  </svg>
                  <input
                    class="PrivateSwitchBase-input-83"
                    data-indeterminate="false"
                    id="colors"
                    name="colors"
                    type="checkbox"
                    value="黄"
                  />
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                黄
              </span>
            </label>
            <label
              class="MuiFormControlLabel-root"
            >
              <span
                aria-disabled="false"
                class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-80 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                    />
                  </svg>
                  <input
                    class="PrivateSwitchBase-input-83"
                    data-indeterminate="false"
                    id="colors"
                    name="colors"
                    type="checkbox"
                    value="緑"
                  />
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                緑
              </span>
            </label>
            <label
              class="MuiFormControlLabel-root"
            >
              <span
                aria-disabled="false"
                class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-80 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                    />
                  </svg>
                  <input
                    class="PrivateSwitchBase-input-83"
                    data-indeterminate="false"
                    id="colors"
                    name="colors"
                    type="checkbox"
                    value="青"
                  />
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                青
              </span>
            </label>
            <label
              class="MuiFormControlLabel-root"
            >
              <span
                aria-disabled="false"
                class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-80 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                    />
                  </svg>
                  <input
                    class="PrivateSwitchBase-input-83"
                    data-indeterminate="false"
                    id="colors"
                    name="colors"
                    type="checkbox"
                    value="紫"
                  />
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </span>
              <span
                class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
              >
                紫
              </span>
            </label>
            <p
              class="MuiFormHelperText-root"
              id="colors"
            >
              好きな色を選択
            </p>
          </div>
        </div>
        <div
          class="MuiBox-root MuiBox-root-156"
        >
          <div
            class="MuiFormControl-root"
          >
            <label
              class="MuiFormLabel-root"
              for="options"
            >
              チェックリスト
            </label>
            <div
              class="MuiFormGroup-root"
            >
              <label
                class="MuiFormControlLabel-root"
              >
                <span
                  class="MuiSwitch-root"
                >
                  <span
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-80 MuiSwitch-switchBase MuiSwitch-colorSecondary"
                  >
                    <span
                      class="MuiIconButton-label"
                    >
                      <span
                        class="MuiSwitch-thumb"
                      />
                      <input
                        class="PrivateSwitchBase-input-83 MuiSwitch-input"
                        id="option1"
                        name="option1"
                        type="checkbox"
                        value="false"
                      />
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </span>
                  <span
                    class="MuiSwitch-track"
                  />
                </span>
                <span
                  class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  option1
                </span>
              </label>
              <label
                class="MuiFormControlLabel-root"
              >
                <span
                  class="MuiSwitch-root"
                >
                  <span
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-80 MuiSwitch-switchBase MuiSwitch-colorSecondary"
                  >
                    <span
                      class="MuiIconButton-label"
                    >
                      <span
                        class="MuiSwitch-thumb"
                      />
                      <input
                        class="PrivateSwitchBase-input-83 MuiSwitch-input"
                        id="option2"
                        name="option2"
                        type="checkbox"
                        value="false"
                      />
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </span>
                  <span
                    class="MuiSwitch-track"
                  />
                </span>
                <span
                  class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  option2
                </span>
              </label>
              <label
                class="MuiFormControlLabel-root"
              >
                <span
                  class="MuiSwitch-root"
                >
                  <span
                    aria-disabled="false"
                    class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-80 MuiSwitch-switchBase MuiSwitch-colorSecondary PrivateSwitchBase-checked-81 Mui-checked"
                  >
                    <span
                      class="MuiIconButton-label"
                    >
                      <span
                        class="MuiSwitch-thumb"
                      />
                      <input
                        checked=""
                        class="PrivateSwitchBase-input-83 MuiSwitch-input"
                        id="option3"
                        name="option3"
                        type="checkbox"
                        value="true"
                      />
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </span>
                  <span
                    class="MuiSwitch-track"
                  />
                </span>
                <span
                  class="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  option3
                </span>
              </label>
            </div>
            <p
              class="MuiFormHelperText-root"
            >
              Be careful
            </p>
          </div>
        </div>
        <div
          class="MuiBox-root MuiBox-root-169"
        >
          <div
            class="MuiFormControl-root"
          >
            <div
              class="MuiFormControl-root MuiTextField-root"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled"
                data-shrink="true"
                for="date"
              >
                公開年月日
              </label>
              <div
                class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              >
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiInput-input"
                  id="date"
                  type="date"
                  value="2017-05-24"
                />
              </div>
            </div>
          </div>
          <div
            class="MuiFormControl-root"
          >
            <div
              class="MuiFormControl-root MuiTextField-root"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled"
                data-shrink="true"
                for="time"
              >
                公開時間
              </label>
              <div
                class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              >
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiInput-input"
                  id="time"
                  step="300"
                  type="time"
                  value="07:30"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiBox-root MuiBox-root-171"
        >
          <div
            class="MuiFormControl-root MuiTextField-root"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled"
              data-shrink="true"
            >
              公開年月日
            </label>
            <div
              class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedEnd"
            >
              <input
                aria-invalid="false"
                class="MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedEnd"
                type="text"
                value="2019/08/28"
              />
              <div
                class="MuiInputAdornment-root MuiInputAdornment-positionEnd"
              >
                <button
                  class="MuiButtonBase-root MuiIconButton-root"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      role="presentation"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M17 12h-5v5h5v-5zM16 1v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2h-1V1h-2zm3 18H5V8h14v11z"
                      />
                      <path
                        d="M0 0h24v24H0z"
                        fill="none"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </div>
            </div>
            
          </div>
          <div
            class="MuiFormControl-root MuiTextField-root"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled"
              data-shrink="true"
            >
              公開時間
            </label>
            <div
              class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl MuiInputBase-adornedEnd"
            >
              <input
                aria-invalid="false"
                class="MuiInputBase-input MuiInput-input MuiInputBase-inputAdornedEnd"
                type="text"
                value="14:46"
              />
              <div
                class="MuiInputAdornment-root MuiInputAdornment-positionEnd"
              >
                <button
                  class="MuiButtonBase-root MuiIconButton-root"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      role="presentation"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M17 12h-5v5h5v-5zM16 1v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2h-1V1h-2zm3 18H5V8h14v11z"
                      />
                      <path
                        d="M0 0h24v24H0z"
                        fill="none"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </div>
            </div>
            
          </div>
        </div>
        <div
          class="MuiBox-root MuiBox-root-198"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
            tabindex="0"
            type="submit"
          >
            <span
              class="MuiButton-label"
            >
              submit
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </form>
  </div>
</body>
`;
